plugin/action/mask/mask.go

```diff
@@ -85,11 +95,10 @@ func factory() (pipeline.AnyPlugin, pipeline.AnyConfig) {
 }
 
 func compileMasks(masks []Mask, logger *zap.SugaredLogger) []Mask {
```

Comment:
@vano144 in this plugins logger passes as arg while mostly used global logger. 
I guess we can use one approach.